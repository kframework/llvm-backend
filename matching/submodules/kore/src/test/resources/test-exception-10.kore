[]

module TEST-EXCEPTION-10

  axiom{}
    #X:#S{#R} []

endmodule []
