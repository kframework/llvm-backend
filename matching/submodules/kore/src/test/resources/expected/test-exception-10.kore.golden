Parse error: Error in module 'TEST-EXCEPTION-10' -> axiom declaration -> \implies (../../../test/resources//test-exception-10.kore 6:8) -> sort '#S' (../../../test/resources//test-exception-10.kore 6:8) -> (../../../test/resources//test-exception-10.kore 6:8): Sort '#S' not declared.